packages/atri-app-core/src/api/canvasApi.ts

```diff
@@ -56,8 +56,11 @@ subscribeCanvasMachine("INSIDE_CANVAS", (_context, event) => {
     "*"
   );
 });
-subscribeCanvasMachine("moveWhileDrag", () => {
-  // TODO: detect catcher & call canvas overlay api
+subscribeCanvasMachine("moveWhileDrag", (context) => {
+  window.parent.postMessage(
+    { type: "moveWhileDrag", id: context.probableParent },
+    "*"
+  );
```

Comment:
Don't post `moveWhileDrag` because sending messages is costly. The `moveWhileDrag` is continuous process and will bombard the message channel.

Reply:
Done